.travis.yml

```diff
@@ -6,11 +6,9 @@ language: cpp
 git:
   depth: false
 
-branches:
-  only:
-  - staging
-  - trying
-  - master
+if: |
+  branch IN (staging, trying, master)
```

Comment:
doesn't this need a slash to continue the line? https://docs.travis-ci.com/user/conditions-v1#line-continuation-multiline-conditions

Reply:
according to [this](https://yaml-multiline.info/) newlines would be kept in the string with `|`

Reply:
However, Travis seems happy...

Reply:
Yeah I think the important bit is "We liked the idea, so we allowed the following:"... FWIW [these](https://github.com/MirServer/mir/blob/master/.travis.yml#L15) have been working for a while.